core/common/src/main/scala/net/liftweb/common/Box.scala

```diff
@@ -456,6 +456,13 @@ sealed abstract class Box[+A] extends Product with Serializable{
    */
   def flatMap[B](f: A => Box[B]): Box[B] = Empty
 
+  def flatten[B](implicit ev: A <:< Box[B]): Box[B] = this match {
```

Comment:
Let's follow the Scala standard library's convention of naming this implicit `asBox`.